forum/errors.py

```diff
@@ -1,69 +1,90 @@
 from rest_framework import status
+from rest_framework.response import Response
 
 
 class Error:
     """Pre-formatted errors for easier handling"""
 
-    class NO_HEADER:
+    class BaseError:
+        msg = None
+        status = None
+
+        @classmethod
+        def response(cls):
+            return Response({'error': cls.msg}, status=cls.status)
```

Comment:
This is a good rule, but now we need to update this response everywhere in the project.

Reply:
Well, it's not neccessery, because you can still get attributes from the Error class directly or not use them at all. I just thought that it would be better to return standart form of the error response authomatically instead of write it manually again and again.